app/views/searches/new.html.erb

```diff
@@ -1,21 +1,11 @@
-<% content_for(:page_title_prefix, "Search") %>
+<% content_for(:page_title_prefix, "Find information on roles") %>
 
-<h2 class="heading-xlarge">Explore work you could do</h2>
+<h2 class="heading-xlarge">Find information on roles</h2>
 
 <% if current_scrapbook.saved_occupations.any? %>
```

Comment:
`SearchesController#new` doesn't have any logic in it, so this isn't duplicated anywhere. You're right that its a bit dirty doing it in the view though – we could move it to the controller? (This hasn't changed as part of this PR though)